welkin/models/calendar.py

```diff
@@ -117,3 +117,28 @@ def get(
         return super().get(
             f"{self._client.instance}/calendar/{route}", params=params, *args, **kwargs
         )
+
+
+class WorkHours(Collection):
+    def get(
+        self,
+        from_date: datetime,
+        to_date: datetime,
+        psm_ids: list = None,
+        timezone: str = None,
```

Comment:
there still needs to be *args and **kwargs here for pagination

Reply:
[e.g.](https://github.com/Lightmatter/welkin-health/blob/c22de759912458e6740ebdbfd0b3a18f39bd4a69/welkin/models/chat.py#L19)